source/docs/networking/networking-introduction/roborio-network-troubleshooting.rst

```diff
@@ -1,61 +1,101 @@
 
+Ping the roboRIO IP address
+---------------------------
+
+If there is no response, try pinging ``10.TE.AM.2`` (:ref:`TE.AM IP Notation <docs/networking/networking-introduction/ip-configurations:TE.AM IP Notation>`) using the command prompt as described above. If this works, you have an issue resolving the mDNS address on your PC. The two most common causes are not having an mDNS resolver installed on the system and a DNS server on the network that is trying to resolve the .local address using regular DNS.
+
+- Verify that you have an mDNS resolver installed on your system. On Windows, this is typically fulfilled by the NI FRC Game Tools. For more information on mDNS resolvers, see the :ref:`roboRIO Networking article <docs/software/roborio-info/roborio-web-dashboard:roboRIO Web Dashboard>`.
+- Disconnect your computer from any other networks and make sure you have the OM5P-AN configured as an access point, using the :ref:`FRC Radio Configuration Utility <docs/zero-to-robot/step-3/radio-programming:Programming your Radio>`. Removing any other routers from the system will help verify that there is not a DNS server causing the issue.
+
+Ping fails
+----------
+
+.. tabs::
+
+   .. group-tab:: Windows 10
+
+      .. image:: images/roborio-troubleshooting/win10-dhcp.png
+         :alt: Windows 10 image of the adapter setting
+
+      If pinging the IP address directly fails, you may have an issue with the network configuration of the PC. The PC should be configured to **Automatic**. To check this, click :guilabel:`Start` -> :guilabel:`Settings` -> :guilabel:`Network & Internet`. Depending on your network, select :guilabel:`Wifi` or :guilabel:`Ethernet`. Then click on your connected network. Scroll down to **IP settings** and click :guilabel:`Edit` and ensure the :guilabel:`Automatic (DHCP)` option is selected.
+
+   .. group-tab:: Windows 7
+
+      .. image:: images/roborio-troubleshooting/control-panel-dhcp.png
+         :alt: Windows 7 image of the adapter setting
+
+      If pinging the IP address directly fails, you may have an issue with the network configuration of the PC. The PC should be configured to **Obtain an Address Automatically** (also known as DHCP). To check this, click :guilabel:`Start` -> :guilabel:`Control Panel` -> :guilabel:`Network Connections` -> :guilabel:`Change adapter settings`, then right click on the appropriate interface (usually Local Area Connection for Ethernet or Wireless Network Connection for wireless) and select :guilabel:`Properties`. Click :guilabel:`Internet Protocol Version 4`, then click :guilabel:`Properties`. Make sure both radio buttons are set to :guilabel:`Obtain automatically`.
+
 USB Connection Troubleshooting
```

Comment:
Why has this been changed to be a sub-topic underneath "Ping Fails"

Reply:
So looking at the previous document I think it suffered from a lot of disjointed thoughts that all got piled in and not under a logical order.  I think we could present these suggestions as distinct troubleshooting steps but I think it wiser to present it as more of a guide that can be walked through based on the previous results of each step.  I think we could rename this but I couldn't think of a situation that the mDNS ping would succeed and the user would really want to go on to the troubleshooting steps below most specifically the ping of the static address.

Reply:
Perhaps you misunderstood. You have this as being under a "category" underneath pinging, when it's not. It's the next step, which should be on an equal level of structure.

Reply:
Ping Fails is a section not a subsection.  What are you saying is should be?

If you are arguing that USB Connection and others should be sections rather than subsections then I think that is fair and I would be open to that.

Reply:
That is precisely what I want :)

Reply:
Done let me know if that didn't solve the issue you had